composer/src/main/kotlin/com/gojuno/composer/Main.kt

```diff
@@ -74,7 +74,7 @@ fun main(rawArgs: Array<String>) {
 
     when {
         totalPassed > 0 && totalFailed == 0 -> exit(Exit.Ok)
-        totalPassed == 0 && totalFailed == 0 -> exit(Exit.NoTests)
+        totalPassed == 0 && totalFailed == 0 -> if(args.failIfNoTests) exit(Exit.NoTests) else exit(Exit.Ok)
```

Comment:
Tests for exit code of main(args) logic will be possible without mock with some abstraction here, let's say 
`fun determineResults(suite: List<Suite>): Exit`

Reply:
yeah, or as an integration test

Reply:
nope